src/components/Notification.tsx

```diff
@@ -0,0 +1,45 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import classNames from 'classnames';
+import React, { useEffect } from 'react';
+
+type Props = {
+  errorMessage: string,
+  setErrorMessage: React.Dispatch<React.SetStateAction<string>>
+};
+
+export const Notification: React.FC<Props> = ({
+  errorMessage,
+  setErrorMessage,
+}) => {
+  const onClick = () => {
+    setErrorMessage('');
+  };
+
+  let timer: number;
+
+  useEffect(() => {
```

Comment:
Add 'errorMessage' and 'setErrorMessage' to the dependencies array of useEffect to handle changes properly